src/ddmd/func.d

```diff
@@ -177,6 +177,76 @@ private bool needsFensure(FuncDeclaration fd)
     return false;
 }
 
+/****************************************************
+ * Rewrite contracts as nested functions, then call them. Doing it as nested
+ * functions means that overriding functions can call them.
+ * Params:
+ *      fd = the function to search
```

Comment:
It's `fdx`

Reply:
`the function to search` ?? What is it searching for?

Reply:
Oops, I didn't update that after copy/paste.